app/views/peeps.erb

```diff
@@ -0,0 +1,13 @@
+<h3> Peeps -</h3>
+
+<% @peeps.each do |peep| %>
+<ul style="list-style-type:disc">
+   User - <%= peep.user.username %> at <%= peep.created_at.strftime("%H:%M, %d/%m/%y") %>
```

Comment:
Two things:

- Can you figure out someway to avoid reaching through `peep` to the user's username? (think about Demeter)
- `strftime(blah)` looks really unpleasant in the view. Can we move this further 'down the stack' to the model somehow, and give us a nicer interface in the view?
- I'm surprised that you chose a `ul` to represent these. What is a `ul` really for?